helm/default-apps-gcp/values.yaml

```diff
@@ -139,3 +139,13 @@ apps:
     forceUpgrade: false
     namespace: kube-system
     version: 1.0.1
+  workload-identity-operator-gcp:
+    appName: workload-identity-operator-gcp
+    chartName: workload-identity-operator-gcp
+    catalog: default
+    clusterValues:
+      configMap: true
+      secret: false
+    forceUpgrade: false
+    namespace: giantswarm
```

Comment:
should we use `kube-system` or `giantswarm`?

Reply:
same as above, and we do this in was as well, run CSI and IRSA in `kube-system`